src/content/nl/2020/accessibility.md

```diff
@@ -0,0 +1,585 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSjkB_XAYiwkYrMuoXp44mdqMGJVDUkTr_48ELozY72Mdv3OlxeWV9ysbDY9bs6hA7LnJTrHar9aZlM/pubchart?oid=1053957382&format=interactive",
+  sheets_gid="1095274901",
+  sql_file="viewport_zoom_scale.sql"
+  )
+}}
+
+We ontdekten dat 29,34% van de desktoppagina's en 30,66% van de mobiele pagina's schalen probeert uit te schakelen door ofwel `maximum-scale` in te stellen op een waarde kleiner dan 1, of `user-scalable` `0` of `none`. Sommige besturingssystemen voldoen niet langer aan de uitgeschakelde zoom- en schaalinstelling in HTML. Voor systemen die het wel respecteren, kan dit de pagina voor sommigen effectief onbruikbaar maken. Zie het artikel van Adrian Roselli, <a hreflang="en" href="https://adrianroselli.com/2015/10/dont-disable-zoom.html"><span lang="en">Don't Disable Zoom</span></a> voor meer informatie over het vermijden van het uitschakelen van browserzoom.
+
+### Taalidentificatie
+
+{{ figure_markup(
+  caption="Desktopsites hebben een geldig kenmerk `lang`.",
+  content="77,67%.",
+  classes="big-number",
+  sheets_gid="812908021",
+  sql_file="02_05.sql"
+)
+}}
+
+Door een HTML-attribuut `lang` in te stellen, kan een pagina gemakkelijk worden vertaald en is er betere ondersteuning voor schermlezers. Het percentage sites met een geldig HTML lang-attribuut op desktop dit jaar was 77,67%, met slechts 77,7% überhaupt een `lang`-attribuut.
+
+## Media op internet
+
+Media is een essentieel onderdeel van de webervaring. Het kan een verrijkte context toevoegen aan de omringende tekstuele informatie, en niet alleen voor ziende gebruikers.
+
+### Afbeeldingen en hun tekstalternatieven
+
+In 1995 introduceerde <a hreflang="en" href="https://www.w3.org/MarkUp/html-spec/html-spec_5.html#SEC5.10">HTML 2.0</a> het `alt`-attribuut, waardoor webauteurs een tekstalternatief konden bieden voor de visuele informatie die in een afbeelding wordt gecommuniceerd. Een schermlezer kan zijn visuele betekenis auditief overbrengen door de alternatieve tekst van de afbeelding aan te kondigen. Als afbeeldingen niet kunnen worden geladen, wordt de alternatieve tekst voor een beschrijving weergegeven.
+
+{{ figure_markup(
+  caption='Mobiele sites passeren de "afbeeldingen met `alt` tekst" Lighthouse audit.',
+  content="54%.",
+  classes="big-number",
+  sheets_gid="580400436",
+  sql_file="lighthouse_a11y_audits.sql"
+)
+}}
+
+Uit de 2020-auditgegevens van Lighthouse blijkt dat slechts 54% van de sites de <a hreflang="en" href="https://dequeuniversity.com/rules/axe/3.5/image-alt">test voor afbeeldingen met `alt`-tekst</a> doorstaat. Deze test zoekt naar de aanwezigheid van ten minste één van de attributen `alt`, `aria-label` en `aria-labelledby` op `img` elementen. In de meeste gevallen is het gebruik van het `alt` -attribuut de beste keuze.
```

Comment:
Just spotted this and not sure if it's even an issue but thought I'd ask the question since, there's a few instances of this. Should the space be there?

```suggestion
Uit de 2020-auditgegevens van Lighthouse blijkt dat slechts 54% van de sites de <a hreflang="en" href="https://dequeuniversity.com/rules/axe/3.5/image-alt">test voor afbeeldingen met `alt`-tekst</a> doorstaat. Deze test zoekt naar de aanwezigheid van ten minste één van de attributen `alt`, `aria-label` en `aria-labelledby` op `img` elementen. In de meeste gevallen is het gebruik van het `alt`-attribuut de beste keuze.
```

And if not should we search and replace any ` -` with `-`?

Reply:
My bad, that is indeed a typo since it should be `-` without any whitespace in between

Reply:
OK. There were a few instances of this.